packages/experiments/src/components/FloatingSuggestionsComposite/FloatingSuggestionsItem/FloatingSuggestionsItem.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import {
+  IFloatingSuggestionItemStylesProps,
+  IFloatingSuggestionItemStyles,
+  IFloatingSuggestionItemProps,
+  IFloatingSuggestionOnRenderItemProps
+} from './FloatingSuggestionsItem.types';
+import { classNamesFunction, css } from '../../../Utilities';
+import { getStyles } from './FloatingSuggestionsItem.styles';
+import { CommandButton, IconButton } from 'office-ui-fabric-react/lib/Button';
+
+export const FloatingSuggestionsItem = <T extends {}>(props: IFloatingSuggestionItemProps<T>): JSX.Element => {
+  const { onClick, className, onRemoveItem, onRenderSuggestion, showRemoveButton, removeButtonAriaLabel, isSelected } = props;
+  const getClassNames = classNamesFunction<IFloatingSuggestionItemStylesProps, IFloatingSuggestionItemStyles>();
+  const classNames = getClassNames(getStyles, { isSelected: isSelected });
+
+  const onClickItem = (ev: React.MouseEvent<HTMLButtonElement, MouseEvent>): void => {
```

Comment:
These all need to be tested... in addition to adding an "Examples.tsx" file, I believe this will need "test.tsx" - using jest, tests should be created.

Reply:
Test cases will be added, just that i'm more focusing on building the functionality first and tests and examples will be added later.

Reply:
Since this is staying in experiments for now, these can be added in a subsequent PR